.github/workflows/sanity.yaml

```diff
@@ -1,12 +1,13 @@
 name: sanity
 
 on:
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
+    - '**/*.md'
```

Comment:
Same thing as above: Let's remove the paths-ignore keyword for now as it doesn't play well with branch protection rules.